pulumi/__main__.py

```diff
@@ -474,9 +483,10 @@ def main():
         connection=connection
     )
 
-    command_copy_justfile = remote.CopyFile(
+    justfile_asset = pulumi.FileAsset("server-side-files/justfile")
```

Comment:
I am curious whether this will still work compared to remote.CopyFile

Reply:
It's working for me, and I removed a warning that Pulumi was generating indicating that remote.CopyFile was deprecated and would be removed.